docs/cluster-management/security.md

```diff
@@ -60,4 +66,6 @@ In order to connect to the operator via the CLI, you must provide valid AWS cred
 
 ## HTTPS
 
-All APIs are accessible via HTTPS. The certificate is autogenerated during installation using `localhost` as the Common Name (CN). Therefore, clients will need to skip certificate verification (e.g. `curl -k`) when using HTTPS.
+All APIs are accessible via HTTPS (in addition to HTTP). The SSL certificate is autogenerated during installation using `localhost` as the Common Name (CN). Therefore, clients will need to skip certificate verification (e.g. `curl -k`) when using HTTPS.
+
+To use AWS's default (trusted) certificate or your own certificate, you can [configure API Gateway](../guides/api-gateway.md) as a proxy to your Cortex cluster. If you do this, you'll also likely want to set Cortex's API load balancer to be internal (versus internet-facing, which is the default); do this by setting `api_load_balancer_scheme: internal` in your [cluster configuration](config.md) file before creating your cluster (e.g. via `cortex cluster up -c cluster.yaml`).
```

Comment:
`If you do this, you'll also likely want to set Cortex's API` I would explain why it is beneficial to set api load balancer to internal.